documentation/Mod_Template/music/music

```diff
@@ -0,0 +1 @@
+file:///C:/Users/Cassius/Downloads/357.%20Icecream%20Truck%20V2.mp3
```

Comment:
Make sure to use the correct code listed below so that your desired music tracks could work in your GXmod.

~~~~~~~~~~~~~~~~~~~~~~~

"tight_package"; {

			"file:///C:/Users/Cassius/Downloads/357.%20Icecream%20Truck%20V2.mp3
",

            "_comment": "The below listed tracks are just for reference,  you can add whatever music track you like and substitute the below given dummy tracks."

			"music/track_2.mp3",
			"music/track_3.mp3",
 
}

~~~~~~~~~~~~~~~~~~~~~~~